pyserini/index/pyutils.py

```diff
@@ -336,3 +336,22 @@ def convert_collection_docid_to_internal_docid(self, docid: str) -> int:
             The Lucene internal ``docid`` corresponding to the external collection ``docid``.
         """
         return self.object.convertDocidToLuceneDocid(self.reader, docid)
+
+    def stats(self) -> dict:
```

Comment:
should we add type hints here?
and below, do some explicit type checking?

Reply:
All keys have type `str` so that would be possible but, the values vary between `int` and `str`, I'm not sure how to hint/type check without the code getting messy..

```python
def stats(self) -> Dict[str, int or str]:
```

Looks kinda strange to me, thoughts?

Reply:
Can the values be string too?

Reply:
Yes, the original `printIndexStats` in Anserini's `IndexUtils` also showed the stored fields as string, e.g. for the 'title' field: 

`(indexOption: DOCS, hasVectors: false)`

Reply:
Hrm, I see.
Yea, `def stats(self) -> dict:` is fine I think.